src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -122,7 +126,12 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
             setSelectedDayIndex(dayIndex);
           }}
         >
-          {formatDayLabel(day)}
+          <button
+            aria-label={WEEKDAYS[day.getDay()]}
```

Comment:
There should be no need for an aria label here given the button already has text on it. And if we do explicitly need an aria-label then it should be using the same text and method of calculating it that we are using for the visible button

Reply:
The visible test is short version. I provided long version as a label.

Reply:
There's a whole bunch of logic that goes into determining what is shown here, based on whether the schedule is timeshifted/etc. We need to ensure whatever we change/add here is going to align with that in all cases; without needlessly increasing the scope of work to maintain changes/avoid bugs.